src/main/java/seedu/address/logic/commands/DoneCommand.java

```diff
@@ -19,19 +22,26 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Marks a task as COMPLETED.\n"
-            + "Parameters: [TASK_ID] \n"
-            + "Example: " + COMMAND_WORD + " 3";
+            + "Parameters: "
+            + "[" + PREFIX_TASK + "TASK_ID] "
+            + "[" + PREFIX_RATING + "RATING]\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TASK + "3 "
+            + PREFIX_RATING + "5";
 
     public static final String MESSAGE_INVALID_ID = "Invalid Task id.";
     public static final String MESSAGE_MARK_TASK_COMPLETED = "Task (TASK ID: %1$s) is marked "
             + TaskStatus.COMPLETED + " successfully.";
     public static final String MESSAGE_TASK_NOT_ONGOING = "Current Task Status: %1$s \n"
             + "Task has to be ONGOING before it can be marked COMPLETED.";
+    public static final String MESSAGE_INVALID_RATING = "Invalid Rating. Rating must be in between 1-5";
```

Comment:
It should be the parser's responsibility to detect an invalid rating.

Reply:
done